model_extension/2comp/dialysis/template/in.2comp.template

```diff
@@ -0,0 +1,142 @@
+# see 1comp template file for the new structure and automation
+# see past (ERROR_) lammps.template file for the old structure
+
+# 3D simulation of VLP1 + VLP2 + Linker system
+units           lj
+boundary        p       p       p
+atom_style      hybrid  charge  sphere
+neighbor        0.3     bin
+neigh_modify    every   1       delay   0       check   yes
+
+## Create Simulation Box, Atoms ##
+read_restart	RestartcUSERHIGHSALT.*  # replace * by number of the latest state
+
+## Group Atoms by Type ##
+group   VLP1    type    1
+group   VLP2    type    2
+group   Linker  type    3
+group   AllVLPS union   VLP1    VLP2    # creates a group with all VLPs
+
+## Defining Particle and Solvent Properties ##
+mass	1	1
+mass	2	1
+mass	3	0.0029024	# relative mass of linker (G6 dendrimer) to VLP (P22)
+dielectric	78.54
+
+## Variables taken from outside script ##
+variable VLP1Charge		equal 	USERVLP1CHARGE				# in e units
+variable VLP2Charge		equal 	USERVLP2CHARGE				# in e units
+variable SIGMAHC1RAW		equal 	USERSIGMAHC1RAW				# in nm
+variable SIGMAHC2RAW		equal 	USERSIGMAHC2RAW				# in nm
+variable SALTCONC		equal 	USERSALTCONC 				# in M
+variable VLPEScutoff		equal	USERVLPEScutoff				# in reduced units
+variable LinkerEScutoff		equal	USERLinkerEScutoff			# in reduced units
+variable VLPLinkerEScutoff	equal	USERVLPLinkerEScutoff			# in reduced units
+variable RUNCOUNT		equal 	10000000				# 10000 short run
+
+## Variables defining interparticle potentials ##
+variable lB		equal 	0.714		# in nm, changes with solvent
+variable VLPDiameterRAW	equal 56		# in nm
+variable LinkerDiameterRAW equal 6.7		# in nm
+variable VLPLJcutoff	equal	2^(1.0/6.0) 	# 2^(1/6) times the VLP diameter 
+
+variable LinkerCharge	equal 	45			# in e units
+variable LinkerDiameter	equal 	v_LinkerDiameterRAW/v_VLPDiameterRAW	# in reduced units adopted by this simulation 
+variable LinkerLJcutoff	equal 	v_LinkerDiameter*(2^(1.0/6.0))	# 2^(1/6) times the linker diameter
+variable SIGMAHC1 	equal 	v_SIGMAHC1RAW/v_VLPDiameterRAW
+variable SIGMAHC2 	equal 	v_SIGMAHC2RAW/v_VLPDiameterRAW
+variable VLP1LinkerLJcutoff	equal v_SIGMAHC1*(2^(1.0/6.0))	# 2^(1/6) times the VLP-linker interaction lengthscale
+variable VLP2LinkerLJcutoff	equal v_SIGMAHC2*(2^(1.0/6.0))	# 2^(1/6) times the VLP-linker interaction lengthscale
+variable DELTA1		equal 	(1+v_LinkerDiameter)*0.5-v_SIGMAHC1	# in reduced units
+variable DELTA2		equal 	(1+v_LinkerDiameter)*0.5-v_SIGMAHC2	# in reduced units
+variable KAPPA		equal 	v_VLPDiameterRAW*sqrt(8*v_lB*v_SALTCONC*PI*0.6022141)	# in reduced units of (1 / vlp diameter)	
+variable VLP1ChargeDLVO	equal	(v_VLP1Charge*exp(v_KAPPA*0.5)/(1+v_KAPPA*0.5))*(1.000676522)	# KAPPA is already in reduced units of the simulation
```

Comment:
The numerical factor at the end of VLP1ChargeDLVO and VLP2ChargeDLVO lines is not matching 1.00065763. The linker charge has this factor and I think 1comp scripts have this factor, so lets make this consistent throughout. The current git 2comp template file has this error fixed. So I am not sure if your PR is asking me to merge older files, which will create potential of bugs getting in?

Reply:
Sorry about that! I'll fix the issue.

Reply:
I'll cancel this request and open a new one